config/data/bsnl_templates.yml

```diff
@@ -78,7 +78,7 @@ hi-IN.notifications.set02.professional_request:
   Max_Length_Permitted: '0'
   Template_Status: '0'
   Is_Unicode: '1'
-hi-IN.notifications.set03.basic:
+hi-IN.notifications.set03.basic: &1
```

Comment:
🍀  This is a YAML feature:  https://docs.geoserver.org/latest/en/user/styling/ysld/reference/variables.html